app/vmalert/group.go

```diff
@@ -99,6 +104,8 @@ func (g *Group) ID() uint64 {
 	hash.Write([]byte(g.File))
 	hash.Write([]byte("\xff"))
 	hash.Write([]byte(g.Name))
+	hash.Write([]byte("\xff"))
```

Comment:
I guess no need in adding token to group ID. The intention is to prevent creating files with the same names and group names no matter which auth tokens they have

Reply:
maybe there is condition like same rules with different tenant setting.

Reply:
AuthToken may be `nil` though and cause a panic